firmware/applications/SignBuddy/main.c

```diff
@@ -117,3 +121,8 @@ void error_handler(void)
     for (uint32_t i = 0; i < 1000000; i++);
   }
 }
+
+void SysTick_Handler(void)
+{
+  HAL_IncTick();
+}
```

Comment:
Is this necessary/used internally by the HAL functions?

Reply:
yeah